packages/boot/test/bootstrapTests/benchmarkerator.js

```diff
@@ -0,0 +1,580 @@
+import process from 'node:process';
```

Comment:
It appears that the benchmarking code and the SDK helper methods are entwined within the same component. For maintainability, reusability, and clarity, I recommend segregating these two functionalities.

Benchmarking Tool (Benchmarkerator): The core tool should primarily deal with orchestrating the benchmark tests, i.e., setting up benchmarks, executing them, and reporting the results. It should be agnostic to the specifics of what it is testing. This will make the tool more generic and possibly usable in other contexts or projects.

BenchmarkTestBase: Instead of having makeSwingsetTestKit and walletFactory actors inside the benchmarking tool, maybe create a separate BenchmarkTestBase that the benchmark implementors import on a need basis.

Reply:
I don't disagree, but we're still working our way down the learning curve on this, and our experience sample size is still to small to yield a clear delineation line.  While there's a lot of stuff that is clearly on one side or the other, there's still enough confused stuff in the middle that I think we need a couple more exemplars in order to get a picture good enough to know where to slice the thing into pieces.